open3d_conversions/include/open3d_conversions/open3d_conversions.hpp

```diff
@@ -41,8 +43,37 @@ namespace open3d_conversions
  */
 void open3dToRos(
```

Comment:
Ref in the comments that the user is expected to fill in the header afterwards with the frame/stamp